### ISSUE TITLE
[FEATURE] Drawer search bar transparency 

### ISSUE THREAD

**Issue by ujwal246** [Deletescape-Media/Lawnchair#4158]
### Describe the feature

Add option to customise drawer search bar transparency to match with less background opacity app drawer.

### Why would it be useful to add?

For Uniform asthetic look.

### Did the feature exist in Lawnchair v2? (Play Store version)

Yes

### Example(s)

![Screenshot_20240328_205931_Lawnchair](https://github.com/LawnchairLauncher/lawnchair/assets/117264003/0c2e3ec0-d7e9-4513-82c0-69fe9bc1d29c)
![Screenshot_20240328_205946_Lawnchair](https://github.com/LawnchairLauncher/lawnchair/assets/117264003/3db8fd39-a839-43aa-8e84-0b729494a09c)

**Comment by charliefrance** [comment]
Yes, the search bar's background needs to match the rest of the drawer.
